core/src/main/java/io/questdb/griffin/engine/functions/finance/LevelTwoPriceFunctionFactory.java

```diff
@@ -147,15 +147,17 @@ public void init(SymbolTableSource symbolTableSource, SqlExecutionContext execut
      * Unrolled loop for 1 pair.
      */
     private static class L2PriceFunction1 extends L2PriceBaseFunction {
-        private final Function size0Func;
-        private final Function targetFunc;
-        private final Function value0Func;
+        private Function size0Func;
+        private Function targetFunc;
+        private Function value0Func;
 
         public L2PriceFunction1(ObjList<Function> args, IntList argPositions) {
             super(args, argPositions);
-            targetFunc = args.getQuick(0);
-            size0Func = args.getQuick(1);
-            value0Func = args.getQuick(2);
+            if (args.size() != 0) {
```

Comment:
This check looks weird. Maybe we could use a special stub function list in `DEFAULT_CLAZZ_VALUES` with many elements to avoid it?